### ISSUE TITLE
Disable „edit object“ for all objects except alias.0 and 0_userdata.0 (without expert mode)

### ISSUE THREAD

**Issue by klein0r** [ioBroker/ioBroker.admin#1560]
At the moment a user can edit every property (and the raw definition) of an object. Many users suggest in the forum or on facebook to edit adapter properties (like common.type) or change common.read / common.write.

**My suggestion is, that all options should be hidden when expert mode is disabled** (except name). They should be available for objects in namespace alias.0 and 0_userdata.0 (where the user can create own objects).

**Comment by Apollon77** [comment]
Userr can also create own onjects in javascript.X and maybe mqtt.X ? (there are some namespaces)

**Comment by Apollon77** [comment]
But I would propose to SHOW the vlaues in non-Expertmode but grey them out (I assume you mean the same)

**Comment by klein0r** [comment]
> Userr can also create own onjects in javascript.X and maybe mqtt.X ? (there are some namespaces)

Yes, by using Scripts or by sending MQTT messages (and not via the admin UI).

> grey them out (I assume you mean the same)

Of course - the user must be able to see the current definition
